docs/docs/config/behaviors.md

```diff
@@ -0,0 +1,100 @@
+| ----------------- | ------------- | ------------------------------------------------------------------------------ | ------------------ |
+| `label`           | string        | Unique label for the node                                                      |                    |
+| `#binding-cells`  | int           | Must be `<2>`                                                                  |                    |
+| `bindings`        | phandle array | A list of two behaviors: one for hold and one for tap                          |                    |
+| `tapping-term-ms` | int           | How long in milliseconds the key must be held to trigger a hold                |                    |
+| `quick-tap-ms`    | int           | Tap twice within this period in milliseconds to trigger a tap, even when held  | -1                 |
+| `flavor`          | string        | Adjusts how the behavior chooses between hold and tap                          | `"hold-preferred"` |
+| `retro-tap`       | bool          | Triggers the tap behavior on release if no other key was pressed during a hold | false              |
+
+The `flavor` property may be one of:
+
+- `"hold-preferred"`
+- `"balanced"`
+- `"tap-preferred"`
+
+See the [hold-tap behavior documentation](/docs/behaviors/hold-tap) for an explanation of each flavor.
+
+| Node  | Behavior                                      |
+| ----- | --------------------------------------------- |
+| `&lt` | [Layer-tap](/docs/behaviors/layers#layer-tap) |
+| `&mt` | [Mod-tap](/docs/behaviors/mod-tap)            |
+
+## Mod-Morph
+
+Creates a custom behavior that triggers one behavior when a key is pressed without
+certain modifiers held or a different one if certain modifiers are held.
+
+### Devicetree
+
+Applies to: `compatible = "zmk,behavior-mod-morph"`
+
+| Property         | Type          | Description                                                                         |
+| ---------------- | ------------- | ----------------------------------------------------------------------------------- |
+| `label`          | string        | Unique label for the node                                                           |
+| `#binding-cells` | int           | Must be `<0>`                                                                       |
+| `bindings`       | phandle array | A list of two behaviors: one for normal press and one for mod morphed press         |
+| `mods`           | int           | A bit field of modifiers which will switch to the morph behavior if any are pressed |
+
+See [dt-bindings/zmk/modifiers.h](https://github.com/zmkfirmware/zmk/blob/main/app/include/dt-bindings/zmk/modifiers.h)
+for a list of modifiers.
+
+You can use the following nodes to tweak the default behaviors:
+
+| Node     | Behavior                                   |
+| -------- | ------------------------------------------ |
+| `&gresc` | [Grave escape](/docs/behaviors/sticky-key) |
```

Comment:
I don't think this link is relevant to grave-escape, though I couldn't find any existing docs that would describe it.

Reply:
Thanks. It looks like I pasted the wrong thing in there. There doesn't appear to be any existing documentation for grave escape.